README.md

```diff
@@ -93,24 +93,30 @@ Find **Protocol Buffers Descriptions** at the [`./pb` directory](./pb).
 
 1. Install tools specified in the previous section (Docker, kubectl, skaffold)
 
-1. Create a Google Kubernetes Engine cluster and make sure `kubectl` is pointing
+2. Create a Google Kubernetes Engine cluster and make sure `kubectl` is pointing
    to the cluster.
 
         gcloud services enable container.googleapis.com
 
         gcloud container clusters create demo --enable-autoupgrade \
-            --enable-autoscaling --min-nodes=3 --max-nodes=10 --num-nodes=5
+            --enable-autoscaling --min-nodes=3 --max-nodes=10 --num-nodes=5 \
+            -- scopes https://www.googleapis.com/auth/cloud_debugger
 
         kubectl get nodes
 
-2. Enable Google Container Registry (GCR) on your GCP project and configure the
+3. Enable Google Container Registry (GCR) on your GCP project and configure the
    `docker` CLI to authenticate to GCR:
 
        gcloud services enable containerregistry.googleapis.com
 
        gcloud auth configure-docker -q
 
-3. Set your project ID on image names:
+4. (OPTIONAL) Allow GCP to access source code via Github. This step is necessary 
```

Comment:
We currently don't explain many things you can get out of this app (like Metrics, Logging, Tracing, Profiling). So I think we should leave this out of the Installation docs.

If you think this should be documented straight away, please create a `docs/using-stackdriver-debugger.md` and write a brief doc that explains how to use it.